ui/app/controllers/settings/tokens.js

```diff
@@ -52,9 +51,11 @@ export default class Tokens extends Controller {
   @action
   verifyToken() {
     const { secret } = this;
+    this.clearTokenProperties();
     const TokenAdapter = getOwner(this).lookup('adapter:token');
 
     this.set('token.secret', secret);
+    this.set('secret', null);
```

Comment:
`thought`: This is interesting because when we dispatch `clearTokenProperties` we're setting the `token` to be `undefined` and then resetting the `secret` to `null` here. How are we thinking about the difference between `null` and `undefined` here?